genesis/engine/solvers/rigid/constraint_solver_decomp.py

```diff
@@ -351,10 +351,181 @@ def add_equality_constraints(self):
                 if self._solver.equality_info[i_e].eq_type == gs.EQUALITY_TYPE.CONNECT:
                     self._func_equality_connect(i_b, i_e)
                 elif self._solver.equality_info[i_e].eq_type == gs.EQUALITY_TYPE.WELD:
-                    pass
+                    self._func_equality_weld(i_b, i_e)
                 elif self._solver.equality_info[i_e].eq_type == gs.EQUALITY_TYPE.JOINT:
                     self._func_equality_joint(i_b, i_e)
 
+    @ti.func
+    def _func_equality_weld(self, i_b, i_e):
```

Comment:
Is there any code duplication compute to "joint" ? If so, it would be great to factorize the code if not too difficult.

Reply:
will try to do that in the future. It's not very easy to merge.